plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -276,11 +276,26 @@ public void tag(String image, String repository, String tag) throws IOException
         doTag(image, repository, tag, dockerDaemonUri);
     }
 
-    public void push(String repository,
-                     String tag,
-                     String registry,
-                     final ProgressMonitor progressMonitor) throws IOException, InterruptedException {
-        doPush(repository, tag, registry, progressMonitor, dockerDaemonUri);
+    /**
+     * Push docker image to the registry
+     *
+     * @param repository
+     *         full repository name to be applied to newly created image
+     * @param tag
+     *         tag of the image
+     * @param registry
+     *         registry url
+     * @param progressMonitor
+     *         ProgressMonitor for images creation process
+     * @return digest of just pushed image
+     * @throws IOException
```

Comment:
Please take a look at the [codestyle spec](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#javadoc) and change method javadoc according to the spec.